### ISSUE TITLE
Out of sync widgets

### ISSUE THREAD

**Issue by luispabon** [Alexays/Waybar#357]
On a 3 display sway setup, the network and screen inhibit widgets can become out of sync. See screenshot below. Screen inhibit is the lock. The current state is connected to wifi and screen inhibit on (lock open).

![image](https://user-images.githubusercontent.com/6388823/58548648-9d785d00-8201-11e9-9816-e6bf63f911b1.png)

Sway 1.1 rc2, Ubuntu 19.04, Waybar c06725a.

**Comment by luispabon** [comment]
Style:
```css
* {
    border:        none;
    border-radius: 0;
    font-family:   "Fira Sans SemiBold";
    font-size:     14px;
    box-shadow:    none;
    text-shadow:   none;
    transition-duration: 0s;
    background-color: rgba(0,0,0,0);
}

window {
    color:      rgba(217, 216, 216, 1);
    background: rgba(35, 31, 32, 0.00);
}

window#waybar.solo {
    background: rgba(35, 31, 32, 0.85);
}

#workspaces {
    margin: 0 5px;
}

#workspaces button {
    padding:    0 5px;
    color:      rgba(217, 216, 216, 0.4);
}

#workspaces button.visible {
    color:      rgba(217, 216, 216, 1);
}

#workspaces button.focused {
    border-top: 3px solid rgba(217, 216, 216, 1);
    border-bottom: 3px solid rgba(217, 216, 216, 0);
}

#workspaces button.urgent {
    color:      rgba(238, 46, 36, 1);
}

#pulseaudio {
    /* font-size: 26px; */
}

#mode, #battery, #temperature, #cpu, #memory, #network, #pulseaudio, #idle_inhibitor, #backlight, #custom-storage, #custom-spotify, #custom-weather, #custom-mail {
    margin:     0px 0px 0px 10px;
    min-width:  25px;
    padding: 0 5px;
    border-top: 3px solid rgba(217, 216, 216, 0.5);
}

#clock {
    margin:     0px 16px 0px 10px;
    min-width:  140px;
}

#battery.warning {
   color:       rgba(255, 210, 4, 1);
}

#battery.critical {
    color:      rgba(238, 46, 36, 1);
}

#battery.charging {
    color:      rgba(217, 216, 216, 1);
}

#custom-storage.warning {
    color:      rgba(255, 210, 4, 1);
}

#custom-storage.critical {
    color:      rgba(238, 46, 36, 1);
}
```

Config:
```json
{
    "layer": "bottom",
    "position": "top",
    "height": 35,
    "modules-left": ["sway/workspaces", "sway/mode"],
    "modules-center": ["clock"],
    "modules-right": ["tray", "custom/spotify", "idle_inhibitor", "custom/weather", "network", "pulseaudio", "temperature", "battery"],
    "sway/mode": {
        "format": " {}"
    },
    "sway/workspaces": {
        "format": "{name}",
        "disable-scroll": true
    },
    "sway/window": {
        "max-length": 80,
        "tooltip": false
    },
    "clock": {
        "format": "{:%a %d %b - %H:%M}",
        "tooltip": false,
        "on-click": "gnome-calendar"
    },
    "battery": {
        "format": "<span font='Font Awesome 5 Free 12'>{icon}</span>  {capacity}% - {time}",
        "format-icons": ["", "", "", "", ""],
        "format-charging": "<span font='Font Awesome 5 Free'></span>  <span font='Font Awesome 5 Free 12'>{icon}</span>  {capacity}% - {time}",
        "format-full":  "<span font='Font Awesome 5 Free'></span>  <span font='Font Awesome 5 Free 12'>{icon}</span>  {capacity}% - Charged",
        "interval": 30,
        "states": {
            "warning": 25,
            "critical": 10
        },
        "tooltip": false
    },
    "network": {
        "format": "{icon}",
        "format-alt": "{ipaddr}/{cidr} {icon}",
        "format-alt-click": "click-right",
        "format-wifi": " {essid} ({signalStrength}%)",
        "format-ethernet": " {ifname}",
        "format-disconnected": "🖧 Disconnected",
        "on-click": "gnome-terminal -e ~/sway/network-manager",
    "tooltip": false
    },
    "pulseaudio": {
        "format": "<span font='Font Awesome 5 Free 12'>{icon:2}</span> {volume:4}%",
        "format-alt": "<span font='Font Awesome 5 Free 12'>{icon:2}</span> {volume:4}%",
        "format-alt-click": "click-right",
        "format-muted": "<span font='Font Awesome 5 Free 12'></span>",
        "format-icons": {
            "phone": [" ", " ", " ", " "],
            "default": ["", "", "", ""]
        },
        "scroll-step": 2,
        "on-click": "pavucontrol",
        "tooltip": false
    },
    "custom/spotify": {
        "interval": 1,
        "return-type": "json",
        "exec": "~/.config/waybar/modules/spotify.sh",
        "exec-if": "pgrep spotify",
        "escape": true
    },
    "custom/storage": {
        "format": "{} ",
        "format-alt": "{percentage}% ",
        "format-alt-click": "click-right",
        "return-type": "json",
        "interval": 60,
        "exec": "~/.config/waybar/modules/storage.sh"
    },
    "backlight": {
        "format": "{icon}",
        "format-alt": "{percent}% {icon}",
        "format-alt-click": "click-right",
        "format-icons": ["", ""],
        "on-scroll-down": "light -A 1",
        "on-scroll-up": "light -U 1"
    },
    "temperature": {
        "format": "<span font='Font Awesome 5 Free 12'></span> {temperatureC:3}°C",
        "hwmon-path": "/sys/devices/platform/coretemp.0/hwmon/hwmon3/temp1_input"
    },
    "custom/weather": {
        "format": "{}",
        "format-alt": "{alt}: {}",
        "format-alt-click": "click-right",
        "interval": 1800,
        "return-type": "json",
        "exec": "~/.config/waybar/modules/weather.sh",
        "exec-if": "ping wttr.in -c1",
        "on-click": "gnome-terminal -e curl -s https://en.wttr.in"
    },
    "idle_inhibitor": {
        "format": "{icon}",
        "format-alt": "{icon} idle {status}",
        "format-alt-click": "click-right",
        "format-icons": {
            "activated": "<span font='Font Awesome 5 Free 11'></span>",
            "deactivated": "<span font='Font Awesome 5 Free 11'></span>"
        },
        "tooltip": false
    },
    "custom/mail": {
        "format": "",
        "format-alt": "{alt} ",
        "format-alt-click": "click-right",
        "interval": 60,
        "return-type": "json",
        "exec": "~/.config/waybar/modules/mail.py",
        "tooltip": false
    },
    "custom/test": {
        "format": "{}",
        "exec": "/tmp/test blub",
        "param": "blah",
        "interval": 5
    },
    "tray": {
        "icon-size": 18
    }
}
```

**Comment by Alexays** [comment]
Screen inhibit is a known bug #296
And for the network part, i'll try to fix that asap

**Comment by luispabon** [comment]
Copy that :+1:

**Comment by Alexays** [comment]
Network fixed on master with b05d4cd413f5f29013557ae21250fa7459abe908

**Comment by luispabon** [comment]
superb, thank you :muscle:
